src/pages/integrations/log-management/applications/activemq.mdx

```diff
@@ -16,7 +16,7 @@ Configure Filebeat using the pre-defined examples below to start sending and ana
   <InstallIntegration/>
   ### Install Filebeat
 
-  <InstallFilebeat />
+  <InstallFilebeat items={["DEB", "RPM"]} />
 
```

Comment:
I think activemq is java, so all os options could be supported, although I doubt any one does

Reply:
For now we will leave as is, until we review again soon